notebooks/test_1.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>This is good but you would need to re-run the cell to update the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/moaatt2/reviewnb-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I have done that you should be able to see it in the changes now.</p>

Reply:
<p>I see that it is done, great.</p>